testing/citest/tests/kube_smoke_test.py

```diff
@@ -389,6 +389,324 @@ def make_deploy_stage(self, imageSource=None, requisiteStages=[], **kwargs):
 
     result.update(kwargs)
     return result
+  def create_statefulset_pipeline(self):
+      bindings = self.bindings
+
+      payload = self.agent.make_json_payload_from_kwargs(
+          job=[{
+              'type': 'manual',
+              'user': '[anonymous]'
+          }],
+          description='Create StatefulSet in Application ' + self.TEST_APP,
+          application=self.TEST_APP)
+      builder = kube.KubeContractBuilder(self.kube_observer)
+      (builder.new_clause_builder('StatefulSet Added', retryable_for_secs=60)
+       .get_resources(
+          'statefulsets', extra_args=[self.TEST_APP+'tstv001', '--namespace', self.TEST_NAMESPACE])
+       .contains_path_value('kind', 'StatefulSet'))
+      return st.OperationContract(
+          self.new_post_operation(
+              title='create_statefulset', data=payload, path='pipelines/' + self.TEST_APP + '/statefulset-pipeline',
+              status_class=st.SynchronousHttpOperationStatus),
+          contract=builder.build())
+
+  def save_statefulset_pipeline(self):
+      deploy_stage = self.make_deploy_stage_statefulset()
+      job = dict(
+          keepWaitingPipelines='false',
+          application=self.TEST_APP,
+          name='statefulset-pipeline',
+          lastModifiedBy='anonymous',
+          limitConcurrent='true',
+          parallel='true',
+          stages=[deploy_stage]
+      )
+      payload = self.agent.make_json_payload_from_kwargs(**job)
+      expect_match = {key: jp.EQUIVALENT(value)
+                      for key, value in job.items()}
+      expect_match['stages'] = jp.LIST_MATCHES(
+          [
+              jp.DICT_MATCHES({key: jp.EQUIVALENT(value)
+                               for key, value in deploy_stage.items()})])
+
+      builder = st.HttpContractBuilder(self.agent)
+      (builder.new_clause_builder('Has Pipeline',
+                                  retryable_for_secs=15)
+       .get_url_path(
+          'applications/{app}/pipelineConfigs'.format(app=self.TEST_APP))
+       .contains_match(expect_match))
+      return st.OperationContract(self.new_post_operation(title='create_statefulset', data=payload, path='pipelines',
```

Comment:
I'm old fasioned and prefer to try to stick to 80 columns if possible. The constraints of github pr column width in the diffs dont help much either having wide columns. If you could break this like
return st.OperationContract(
    self.new_post_operation(...),
    contract=builder.build())
if might fit. Otherwise you might need to wrap the post at the "status_class" as well.

This is true of all the simliar lines for other contracts. I think usually breaking wrapping at the "path" argument will be desirable because the other path expressions are long (this one isnt).

Reply:
Code changed , as per you comments